server/server.go

```diff
@@ -465,6 +436,42 @@ func New(ctx context.Context, config *Config) (*Server, error) {
 	return s, nil
 }
 
+func (s *Server) getHostIP(configIP string) (net.IP, error) {
+	// emulate kubelet behavior of choosing hostIP
+	// ref: k8s/pkg/kubelet/nodestatus/setters.go
+
+	// use configured value if set
+	if hostIP := net.ParseIP(configIP); hostIP != nil {
```

Comment:
I would error here in case it is not parseable to give the user the feedback that CRI-O is misconfigured. Or validate this one in the config validation part of the `config` package. (At least in the forward-port)